streaming/consumer.cc

```diff
@@ -75,6 +78,9 @@ mutation_reader_consumer make_streaming_consumer(sstring origin,
                                 sstable_list_to_mark_as_repaired->insert(loading_sst);
                             }
                         }
+                        if (loading_sst == sst) {
+                            co_await loading_sst->seal_sstable(cfg.backup);
+                        }
```

Comment:
Shouldn't add_new_sstable_and_update_cache() seal it? Since it seals the split ones, it makes sense if it also seals the original one if there's no need to split.

Reply:
Ok.